source/_components/pilight.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Pilight"
+description: "Instructions how to setup MQTT within Home Assistant."
+date: 2015-08-07 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: pilight.png
+ha_category: Hub
+featured: true
+ha_release: pre 0.26
```

Comment:
`0.25` is fine. We just used `pre` before 0.7 as we didn't have a proper versioning schema at that time.